File: 백준/Bronze/5086. 배수와 약수/배수와 약수.py
A, B = map(int, input().split())

while A != 0 and B != 0:
    if A > B:
        if A % B == 0:
            print("multiple")
        else:
            print("neither")
    else:
        if B % A == 0:
            print("factor")
        else:
            print("neither")
    A, B = map(int, input().split())

File: 백준/Bronze/5086. 배수와 약수/배수와 약수.java
import java.io.*;
import java.util.*;

public class Main {
	public static void main(String[] args) throws IOException {
		Scanner sc = new Scanner(System.in);
		int A = sc.nextInt();
		int B = sc.nextInt();

		while (A != 0 && B != 0) {
			if (A > B) {
				if (A % B == 0) {
					System.out.println("multiple");
				} else {
					System.out.println("neither");
				}
			} else {
				if (B % A == 0) {
					System.out.println("factor");
				} else {
					System.out.println("neither");
				}
			}
			A = sc.nextInt();
			B = sc.nextInt();
		}
		sc.close();
	}
}